server/sonar-server/src/main/java/org/sonar/server/ce/ws/ActivityAction.java

```diff
@@ -216,22 +217,28 @@ private CeTaskQuery buildQuery(DbSession dbSession, ActivityWsRequest request) {
       query.setStatuses(request.getStatus());
     }
 
-    loadComponentUuids(dbSession, request, query);
+    query.setComponentUuids(loadComponentUuids(dbSession, request));
```

Comment:
Small but good improvement ! Having a method called "loadComponentUuids" that update the query is bad...